bokeh/models/tiles.py

```diff
@@ -90,3 +94,12 @@ class BBoxTileSource(MercatorTileSource):
     use_latlon = Bool(default=False, help="""
     Flag which indicates option to output {XMIN},{YMIN},{XMAX},{YMAX} in meters or latitude and longitude.
     """)
+
+# tile providers -----------------------------------------------------------------
+stamen_toner = WMTSTileSource(url='http://tile.stamen.com/toner/{Z}/{X}/{Y}.png', attribution="""
+Map tiles by <a href="http://stamen.com">Stamen Design</a>, under <a href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a>. Data by <a href="http://openstreetmap.org">OpenStreetMap</a>, under <a href="http://www.openstreetmap.org/copyright">ODbL</a>""")  # NOQA
+
+stamen_toner_labels = WMTSTileSource(url='http://tile.stamen.com/toner-labels/{Z}/{X}/{Y}.png', attribution="""
+Map tiles by <a href="http://stamen.com">Stamen Design</a>, under <a href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a>. Data by <a href="http://openstreetmap.org">OpenStreetMap</a>, under <a href="http://www.openstreetmap.org/copyright">ODbL</a>""")  # NOQA
+
+stamen_terrain = WMTSTileSource(url='http://tile.stamen.com/terrain/{Z}/{X}/{Y}.png', attribution="""Map tiles by <a href="http://stamen.com">Stamen Design</a>, under <a href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a>. Data by <a href="http://openstreetmap.org">OpenStreetMap</a>, under <a href="http://creativecommons.org/licenses/by-sa/3.0">CC BY SA</a>.""")  # NOQA
```

Comment:
Two small things:

These should not go here, they should go in `bokeh.standard_tiles` or something. The models modules should be fairly "clean" and just contain models. 

And also these are long lines even by my standards :) I'd suggest a formatting more like:
```
stamen_toner = WMTSTileSource(
    url='http://tile.stamen.com/toner/{Z}/{X}/{Y}.png', 
    attribution=(
        "Map tiles by <a href="http://stamen.com">Stamen Design</a>, "
        "under <a href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a>. "
        "Data by <a href="http://openstreetmap.org">OpenStreetMap</a>, "
        "under <a href="http://www.openstreetmap.org/copyright">ODbL</a>"
    )
) 

stamen_toner_labels = WMTSTileSource(
    url='http://tile.stamen.com/toner-labels/{Z}/{X}/{Y}.png', 
    attribution=(
        "Map tiles by <a href="http://stamen.com">Stamen Design</a>, "
        "under <a href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a>. " 
        "Data by <a href="http://openstreetmap.org">OpenStreetMap</a>, "
        "under <a href="http://www.openstreetmap.org/copyright">ODbL</a>"
    )
)  

stamen_terrain = WMTSTileSource(
    url='http://tile.stamen.com/terrain/{Z}/{X}/{Y}.png', 
    attribution=(
        "Map tiles by <a href="http://stamen.com">Stamen Design</a>, "
        "under <a href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a>. "
        "Data by <a href="http://openstreetmap.org">OpenStreetMap</a>, "
        "under <a href="http://creativecommons.org/licenses/by-sa/3.0">CC BY SA</a>."
    )
) 
```